python/ray/_private/utils.py

```diff
@@ -1852,3 +1852,36 @@ def __exit__(self, exc_type, exc, exc_tb):
             # If exception was raised in context, returning False will cause it to be
             # reraised.
             return False
+
+
+background_tasks = set()
+
+
+def run_background_task(coroutine: Coroutine, *args, **kwargs) -> asyncio.Task:
+    """Schedule a task reliably to the event loop.
+
+    This API is used when you don't want to cache the reference of `asyncio.Task`.
+    For example,
+
+    ```
+    get_event_loop().create_task(coroutine(*args))
+    ```
+
+    The above code doesn't guarantee to schedule the coroutine to the event loops
+
+    When using create_task in a  "fire and forget" way, we should keep the references
+    alive for the reliable execution. This API is used to fire and forget
+    asynchronous execution.
+
+    https://docs.python.org/3/library/asyncio-task.html#creating-tasks
+    """
+    task = get_or_create_event_loop().create_task(coroutine(*args, **kwargs))
+    global background_tasks
```

Comment:
btw we don't this global here because we aren't setting the value here. Just calling methods.